docs/CHANGELOG.asciidoc

```diff
@@ -68,6 +68,427 @@ Review important information about the {kib} 7.x releases.
+* Refactored policies list to use EuiInMemoryTable {kibana-pull}107510[#107510]
+* Transforms: Adds a type column to the transforms management table {kibana-pull}106990[#106990]
+* Adds a flyout with linked index templates {kibana-pull}106734[#106734]
+* Adds es version field support {kibana-pull}104870[#104870]
+* Adds preview for runtime fields {kibana-pull}100198[#100198]
+Maps::
+* 'show this layer only' layer action {kibana-pull}107947[#107947]
+* Adds indication in layer TOC when layer is filtered by map bounds {kibana-pull}107662[#107662]
+* Show actionable message when term joins have no matches {kibana-pull}105161[#105161]
+* Adds edit tools defaults for user and timestamp {kibana-pull}103588[#103588]
+* Auto generate legends and styles from mvt data {kibana-pull}94811[#94811]
+Metrics::
+* Moves saved views button to page header {kibana-pull}107951[#107951]
+* Adds manage rules link to alerts dropdown {kibana-pull}107950[#107950]
+* Adds checkbox to optionally drop partial buckets from threshold alerts {kibana-pull}107676[#107676]
+* Adds system.cpu.total.norm.pct to default metrics {kibana-pull}102428[#102428]
+Monitoring::
+* Enables OOTB alerts in RAC page and multiple rules of a rule type {kibana-pull}106457[#106457]
+Osquery Manager::
+* Fixes scheduled query status {kibana-pull}106600[#106600]
+Platform::
+* Improves not found response handling in the saved objects repository {kibana-pull}108749[#108749]
+* Updates `esaggs` expressions function to return partial results {kibana-pull}105620[#105620]
+* Updates expressions public API to expose partial results support {kibana-pull}102403[#102403]
+* Changes execution of alerts from async to sync {kibana-pull}97311[#97311]
+Querying & Filtering::
+* Filter FilterBar suggestions by time (according to flag) {kibana-pull}107192[#107192]
+Reporting::
+* Adds support of chunked export {kibana-pull}108485[#108485]
+* Consolidate report job warnings and add warning for deprecated types {kibana-pull}106184[#106184]
+* Adds warning logs about CSV export type being deprecated {kibana-pull}104025[#104025]
+* New UI for migrating reporting indices ILM policy {kibana-pull}103853[#103853]
+Security::
+* Support authenticating to Elasticsearch via service account tokens {kibana-pull}102121[#102121]
+* Space management page UX improvements {kibana-pull}100448[#100448]
+Sharing::
+* Redirect endpoint {kibana-pull}103899[#103899]
+* Adds *Lens* markdown plugin {kibana-pull}96703[#96703]
+Uptime::
+* Improve dedupe client performance {kibana-pull}103979[#103979]
+* Multi Series View {kibana-pull}103855[#103855]
+* Adds browser monitors configuration options {kibana-pull}102928[#102928]
+Other::
+* Adds Thumbnails to Search UI {kibana-pull}104199[#104199]
+
+[float]
+[[fixes-v7.15.0]]
+=== Bug Fixes
+APM::
+* Custom links creation don't work {kibana-pull}110676[#110676]
+* Show relevant nodes in focused service map {kibana-pull}108028[#108028]
```

Comment:
```suggestion
* Show relevant nodes in focused service map {kibana-pull}108028[#108028]
* Display throughput as tps (instead of tpm) when bucket size < 60 seconds {kibana-pull}107850[#107850]
```